### ISSUE TITLE
Deprecation warnings when starting AppDaemon 4.0.0b1

### ISSUE THREAD

**Issue by electrofloat** [home-assistant/appdaemon#768]
As the title says some deprecation warnings when starting AppDaemon 4.0.0b1:
```
Oct 19 14:44:19 localhost python3[28372]: /home/homeassistant/appdaemon-4/lib/python3.6/site-packages/appdaemon/http.py:276: DeprecationWarning: Application.make_handler(...) is deprecated, use AppRunner API instead
Oct 19 14:44:19 localhost python3[28372]:   handler = self.app.make_handler()
Oct 19 14:44:19 localhost python3[28372]: /home/homeassistant/appdaemon-4/lib/python3.6/site-packages/appdaemon/plugins/hass/hassplugin.py:541: DeprecationWarning: verify_ssl is deprecated, use ssl=False instead
Oct 19 14:44:19 localhost python3[28372]:   r = await self.session.get(apiurl, headers=headers, verify_ssl=self.cert_verify)
Oct 19 14:44:19 localhost python3[28372]: /home/homeassistant/appdaemon-4/lib/python3.6/site-packages/appdaemon/plugins/hass/hassplugin.py:462: DeprecationWarning: verify_ssl is deprecated, use ssl=False instead
Oct 19 14:44:19 localhost python3[28372]:   r = await self.session.post(apiurl, headers=headers, json=data, verify_ssl=self.cert_verify)
```

checking the dev branch it seems that the first warning was fixed here: f9954657
but not the other 2.

**Comment by acockburn** [comment]
What version of aiohttp are you using? As you said, the first has been fixed, but I haven't seen the other 2 despite using several versions of python. There will be some work in this area before the 4.0.0 release so we will address it, just curious that I haven't seen that message at all.

**Comment by electrofloat** [comment]
```
(appdaemon-4) homeassistant@hass:~/appdaemon-4$ pip show aiohttp
Name: aiohttp
Version: 3.4.4
```
Unfortunately I'm not familiar how pip handles packages and versions, but checking the website (https://pypi.org/project/aiohttp/#history) the latest version is 3.6.2, so I don't know why a freshly installed appdaemon came with 3.4.4

**Comment by acockburn** [comment]
This should be fixed in 4.0.0, just about to be released  - we upgraded to the latest version of aiohttp
